spring-integration-core/src/test/java/org/springframework/integration/support/management/micrometer/MicrometerMetricsTests.java

```diff
@@ -264,7 +275,11 @@ public void testMicrometerMetrics() {
 
 		@Bean
 		public static MeterRegistry meterRegistry() {
-			return new SimpleMeterRegistry();
+			SimpleMeterRegistry registry = new SimpleMeterRegistry();
+			registry.config().meterFilter(MeterFilter.deny(id ->
+					"channel".equals(id.getTag("type")) &&
+					"noMeters".equals(id.getTag("name"))));
```

Comment:
Probably writing this test you were going to add something to docs.
Aren't you?